applications/FluidTransportApplication/custom_python/fluid_transport_python_application.cpp

```diff
@@ -48,9 +48,12 @@ PYBIND11_MODULE(KratosFluidTransportApplication, m)
 
 	//registering variables in python
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,PECLET);
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,COURANT);
```

Comment:
There is already a variable in the core for the Courant number.

Reply:
BTW, its in the cfd_variables.h

Reply:
Thank you.